ajv/tests/schemas/Invalid/test_invalid_new_routing_and.json

```diff
@@ -0,0 +1,179 @@
+{
```

Comment:
For consistency, the dir should be lower case, ie `invalid`.

Reply:
👍